### PR TITLE
Proposal: Kyverno to Graduation

### PR DESCRIPTION
**Pull request by realshuting**
Hello TOC and CNCF community  👋

We are proposing Kyverno to become a CNCF Graduated project

## What is Kyverno?
[Kyverno](https://kyverno.io/) is a Kubernetes policy engine that was created to validate, mutate, generate, and cleanup Kubernetes resources, including custom resources. Kyverno also offers built-in image verification rules to help secure the software supply chain by verifying signatures and attestations in either Sigstore Cosign and CNCF Notation formats.

## Why are we ready to graduate?
Kyverno graduated to Incubating status in July 2022 and has seen steady [growth](https://kyverno.devstats.cncf.io/d/81/community-health?orgId=1&var-repo_name=Kyverno&var-metric=Stargazers&var-table=swatchers&var-pref=&var-met1=watch&var-met2=watch&from=1656604800000&to=now) of its community. The GitHub stars for Kyverno itself has experienced an impressive growth from 2537 to [4700](https://github.com/kyverno/kyverno), and there are 2,500+ members registered for the [Kyverno community Slack](https://main.kyverno.io/community/#slack-channel). The Kyverno organization has [1452](https://kyverno.devstats.cncf.io/d/18/overall-project-statistics-table?orgId=1&var-period_name=Last%20decade&var-repogroup_name=All) contributors from [280+](https://kyverno.devstats.cncf.io/d/5/companies-table?orgId=1&var-period_name=Last%20decade&var-metric=contributions) companies. Kyverno is used in production and at scale by several end-user organizations like Deutsche Telekom, Spotify, The US Department of Defense, LinkedIn, Vodafone, and Yahoo.

Kyverno completed a [fuzzing security audit](https://kyverno.io/blog/2023/09/06/kyverno-completes-fuzzing-security-audit/) and a [third-party security review](https://kyverno.io/blog/2023/11/28/kyverno-completes-third-party-security-audit/) in November 2023.

The project maintainers believe that Kyverno meets the requirements for Graduation.

## How you can help today
If you love Kyverno, please show your support with a 👍, ❤️ , or 🎉  on this PR!
If you have adopted Kyverno, please add your story to [Adopter.md](https://github.com/kyverno/kyverno/blob/main/ADOPTERS.md).

### CODE REVIEW

**Comment by KRRK2015** [comment]
This is long overdue.... Go Kyverno....

**Comment by rohitg00** [comment]
All the best.

**Comment by anusha94** [comment]
Go Kyverno 🚀

**Comment by eddycharly** [comment]
\+ :100: !

**Comment by melshakir** [comment]
Yay - this is awesome!!

**Comment by rachnasud** [comment]
Way to go Kyverno.

**Comment by Namanl2001** [comment]
onwards and upwards 🚀

**Comment by CIPHERTron** [comment]
+1 Go Kyverno 🚀

**Comment by anubhav888** [comment]
+1 Go Kyverno!

**Comment by cdavisafc** [comment]
+1 YES!

**Comment by vponoikoait** [comment]
+1, great tool!

**Comment by dzolotusky** [comment]
hi @realshuting - @angellk and I will sponsor Kyverno's application to Graduation.

The next step for us is to schedule a kick off meeting as described in https://github.com/cncf/toc/blob/main/process/README.md#application-kick-off-meeting-is-scheduled-and-held. We'll need to go through the new process as outlined in that README file.

File: .github/ISSUE_TEMPLATE/graduation.md [hunk 1]
```diff
@@ -0,0 +1,387 @@
...
+
+<!-- (Project assertion goes here) --> 
+
+[Maintainers](https://github.com/kyverno/kyverno/blob/main/MAINTAINERS.md).
+
+- [x] **A number of active maintainers which is appropriate to the size and scope of the project.**
+
+<!-- (Project assertion goes here) --> 
+
+The Kyverno project currently has eight (8) maintainers from four (4) affiliations.
```
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/graduation.md:120]:
```suggestion
The Kyverno project currently has five (5) maintainers from one (1) affiliation.
```

File: .github/ISSUE_TEMPLATE/graduation.md [hunk 2]
```diff
@@ -0,0 +1,387 @@
...
+The process of onboarding a maintainer is initiated by submitting a pull request. 
+
+For example, [feat: add myself (vishal-chdhry) to maintainers list #9125](https://github.com/kyverno/kyverno/pull/9125).
+
+The same with offboarding process, and emeritus maintainers are listed at: 
+
+https://github.com/kyverno/kyverno/blob/main/MAINTAINERS.md#maintainers-emeritus
+
+
+- [x] **Project maintainers from at least 2 organizations that demonstrates survivability.**
```
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/graduation.md:141]:
```suggestion
- [ ] **Project maintainers from at least 2 organizations that demonstrates survivability.**
```

File: .github/ISSUE_TEMPLATE/graduation.md [hunk 3]
```diff
@@ -0,0 +1,387 @@
...
+
+<!-- (Project assertion goes here) --> 
+
+Currently maintainers are from four (4) organizations including Nirmata, Stackwatch, Ohio Supercomputer Center, Giant Swarm GmbH.
```
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/graduation.md:145]:
```suggestion
Currently maintainers are from one (1) organization, Nirmata.
```

File: .github/ISSUE_TEMPLATE/graduation.md [hunk 4]
```diff
@@ -0,0 +1,387 @@
...
+- [ ] **All subprojects, if any, are listed.**
+
+<!-- (Project assertion goes here) --> 
+
+
+- [x] **If the project has subprojects: subproject leadership, contribution, maturity status documented, including add/remove process.**
+
+<!-- (Project assertion goes here) --> 
+
+This is clarified in the [Project Governance section](https://main.kyverno.io/community/#project-governance).
```
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/graduation.md:175]:
```suggestion
This is clarified in the [Project Governance section](https://kyverno.io/community/#project-governance).
```

File: .github/ISSUE_TEMPLATE/graduation.md [hunk 5]
```diff
@@ -0,0 +1,387 @@
...
+<!-- (Project assertion goes here) --> 
+
+The strategies for major Project Governance were discussed and refined during the Kyverno maintainers' meeting. You can find the meeting notes [here](https://docs.google.com/document/d/1I_GWsz32gLw8sQyuu_Wv0-WQrtRLjn9FuX2KGNkvUY4/edit):
+* 05-21-2024
+* 02-20-2024
+* 12-05-2023
+
+The Governance page has been maintained and updated through Kyverno website's Github repo, history:
+
+https://github.com/kyverno/website/commits/main/content/en/community/_index.md
```
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/graduation.md:79]:
```suggestion
https://kyverno.io/community/
```

File: .github/ISSUE_TEMPLATE/graduation.md [hunk 6]
```diff
@@ -0,0 +1,387 @@
...
+
+### Required
+
+- [ ] **Clear and discoverable project governance documentation.**
```
Comment (chipzoller) [inline .github/ISSUE_TEMPLATE/graduation.md:83]:
```suggestion
- [x] **Clear and discoverable project governance documentation.**
```

**Review by chipzoller — CHANGES_REQUESTED** [review]
Sorry for such a late review here.

**Comment by angellk** [comment]
The TOC has requested a review be performed by TAG Contributor Strategy to ascertain the sustainability and direction of the project based on its documented and practiced governance. cc: @dzolotusky

**Comment by JimBugwadia** [comment]
> The TOC has requested a review be performed by TAG Contributor Strategy to ascertain the sustainability and direction of the project based on its documented and practiced governance. cc: @dzolotusky

Requested at: https://github.com/cncf/tag-contributor-strategy/issues/696

File: .github/ISSUE_TEMPLATE/graduation.md (hunk 4 above)
Comment (realshuting) [reply .github/ISSUE_TEMPLATE/graduation.md:175]: The project governance has been moved to `kyverno/community` GitHub repo, updated to https://github.com/kyverno/community/blob/main/GOVERNANCE.md.

File: .github/ISSUE_TEMPLATE/graduation.md (hunk 6 above)
Comment (realshuting) [reply .github/ISSUE_TEMPLATE/graduation.md:83]: Nice catch, updated!

File: .github/ISSUE_TEMPLATE/graduation.md (hunk 5 above)
Comment (realshuting) [reply .github/ISSUE_TEMPLATE/graduation.md:79]: Pointed to the `kyverno/community` repo.

**Review by JimBugwadia — APPROVED** [review]

**Comment by angellk** [comment]
Adopter interview 3/6 completed cc: @dzolotusky

**Comment by angellk** [comment]
Adopter interview 4/6 completed, cc: @dzolotusky

**Comment by JimBugwadia** [comment]
@dzolotusky @angellk - thank you for your work on this! The transitions were unexpected, but we are making good progress in on-boarding new maintainers, which we believe will lead to a stronger community. We will re-visit, as soon as the process is back in eligibility status.

**Pull request closed without merging**